File: Python + CPython + Flask/Aggregate.py
from Event import Event

class Aggregate:

    def __init__(self, aggregate_uuid):
        self.aggregate_uuid = aggregate_uuid
        self.state = ""

    def apply(self, event):
        self.state += event.data

    def load_from_events(self):
        events = Event.query.filter_by(aggregate_uuid=self.aggregate_uuid).all()
        for event in events:
            self.apply(event)

    def get_state(self):
        return self.state

File: TypeScript + Deno + Express/src/aggregate.ts
import { Event } from "./entities/event.ts";

class Aggregate {
  private aggregateUuid: string;
  private state: string = "";

  constructor(aggregateUuid: string) {
    this.aggregateUuid = aggregateUuid;
  }

  apply(event: Event): void {
    this.state += event.data;
  }

  getAggregateUuid(): string {
    return this.aggregateUuid;
  }

  getState(): string {
    return this.state;
  }
}

export { Aggregate };
